docs/RFCS/online_schema_change.md

```diff
@@ -131,8 +131,24 @@ Since schema change operations are potentially long running they need
 to be restartable or abortable if the node performing them dies. We
 accomplish this by performing the schema change operation for a table
 on a well known node: the replica holding the leader lease for the
```

Comment:
Or the gossip callback on the node can check whether there's an active leader for each table it owns. That's going to be better actually (though it'll race for the lease, but who cares).